app/(main)/workflows/components/graph.tsx

```
@@ -0,0 +1,342 @@
+      <div className="bg-orange-50 p-3 border-b border-gray-200">
+        <div className="flex items-center">
+          <AlertTriangle className="w-5 h-5 mr-2 text-orange-500" />
+          <h3 className="font-semibold text-gray-800">Trigger</h3>
+        </div>
+      </div>
+      <div className="p-4">
+        <Tooltip>
+          <TooltipTrigger className="space-y-2">
+            {data.selected_triggers.map((current: string, index: number) =>
+              index == 2 ? (
+                <>+{data.selected_triggers.length - 2}</>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)